Exception while notifying

Happened after installing https://plugins.jetbrains.com/plugin/22315-sumnekolua and https://plugins.jetbrains.com/plugin/30908-ansible-lsp

```
Exception while notifying `com.intellij.openapi.extensions.impl.ExtensionPointImpl$doAddChangeListener$listenerAdapter$1@40d23b63`

com.intellij.diagnostic.PluginException: Argument for @NotNull parameter 'mappings' of com/redhat/devtools/lsp4ij/LanguageServersRegistry.addServerDefinitionWithoutNotification must not be null [Plugin: com.redhat.devtools.lsp4ij]
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:58)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance(ServiceInstanceInitializer.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:181)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndspatched(Undispatched.kt:67)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:43)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:179)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:20)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:360)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:134)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:53)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:164)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:29)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:154)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:112)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext$suspendImpl(LazyInstanceHolder.kt:104)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext(LazyInstanceHolder.kt)
	at com.intellij.serviceContainer.ComponentManagerImplKt$doGetOrCreateInstanceBlocking$1.invokeSuspend(ComponentManagerImpl.kt:1698)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:112)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$BuildersKt__BuildersKt(Builders.kt:87)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlockingWithParallelismCompensation(Builders.kt:63)
	at kotlinx.coroutines.BuildersKt.runBlockingWithParallelismCompensation(Unknown Source)
	at kotlinx.coroutines.internal.intellij.IntellijCoroutines.runBlockingWithParallelismCompensation(intellij.kt:48)
	at com.intellij.util.IntelliJCoroutinesFacade.runBlockingWithParallelismCompensation(IntelliJCoroutinesFacade.kt:35)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization$lambda$0(ComponentManagerImpl.kt:1829)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext$lambda$0(context.kt:96)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext(context.kt:95)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization(ComponentManagerImpl.kt:1814)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doGetOrCreateInstanceBlocking(ComponentManagerImpl.kt:1697)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1686)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:798)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:736)
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.getInstance(LanguageServersRegistry.java:64)
	at com.redhat.devtools.lsp4ij.features.semanticTokens.viewProvider.LSPSemanticTokensStructurelessFileViewProviderRegistrar$LanguageServerDefinitionsChangedListener.registerOnce(LSPSemanticTokensStructurelessFileViewProviderRegistrar.java:135)
	at com.redhat.devtools.lsp4ij.features.semanticTokens.viewProvider.LSPSemanticTokensStructurelessFileViewProviderRegistrar$ProjectOpenedListener.execute(LSPSemanticTokensStructurelessFileViewProviderRegistrar.java:112)
	at com.intellij.ide.startup.impl.StartupManagerImpl$Companion$addActivityEpListener$1$extensionAdded$1.invokeSuspend(StartupManagerImpl.kt:113)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
	Suppressed: com.intellij.util.ExceptionUtilRt$RethrownStack: Rethrown at
		at com.intellij.util.ExceptionUtilRt.addRethrownStackAsSuppressed(ExceptionUtilRt.java:43)
		at com.intellij.util.ExceptionUtilRt.rethrowUnchecked(ExceptionUtilRt.java:17)
		at com.intellij.util.ExceptionUtil.rethrowUnchecked(ExceptionUtil.java:143)
		at com.intellij.util.ExceptionUtil.rethrow(ExceptionUtil.java:158)
		at com.intellij.util.ConcurrencyUtil.manifestExceptionsIn(ConcurrencyUtil.java:329)
		at com.intellij.codeInsight.daemon.impl.DaemonCodeAnalyzerImpl.lambda$queuePassesCreation$4(DaemonCodeAnalyzerImpl.java:1381)
		at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:246)
		at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
		at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:208)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:343)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:341)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:710)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1098)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:652)
		at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:307)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:650)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:199)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
		at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1$0(NonBlockingFlushQueue.kt:329)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:941)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1(NonBlockingFlushQueue.kt:328)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:325)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:753)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:321)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:158)
		at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
		at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:731)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:384)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1122)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1122)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:382)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:422)
		at com.intellij.ide.IdeEventQueue.flushQueue$lambda$0(IdeEventQueue.kt:749)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.ide.IdeEventQueue.flushQueue(IdeEventQueue.kt:745)
		at com.intellij.ide.plugins.newui.PluginDetailsPageComponent.showPlugin$lambda$2(PluginDetailsPageComponent.kt:1136)
		at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:211)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:343)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:341)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:710)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1098)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:652)
		at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:307)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:650)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:199)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
		at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1$0(NonBlockingFlushQueue.kt:329)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:941)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1(NonBlockingFlushQueue.kt:328)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:325)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:753)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:321)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:158)
		at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
		at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:731)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:384)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1122)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1122)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:382)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:422)
		at com.intellij.ide.IdeEventQueue.flushQueue$lambda$0(IdeEventQueue.kt:749)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.ide.IdeEventQueue.flushQueue(IdeEventQueue.kt:745)
		at com.intellij.ide.plugins.newui.PluginDetailsPageComponent.showPlugin$lambda$2(PluginDetailsPageComponent.kt:1136)
		at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:211)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:343)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:341)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:710)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1098)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:652)
		at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:307)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:650)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:199)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
		at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1$0(NonBlockingFlushQueue.kt:329)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:941)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1(NonBlockingFlushQueue.kt:328)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:325)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:753)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:321)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:158)
		at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
		at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:731)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:384)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1122)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1122)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:382)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:422)
		at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
		at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: com.intellij.util.ExceptionUtilRt$RethrownStack: Rethrown at
		at com.intellij.util.ExceptionUtilRt.addRethrownStackAsSuppressed(ExceptionUtilRt.java:43)
		at com.intellij.util.ExceptionUtilRt.rethrowUnchecked(ExceptionUtilRt.java:17)
		at com.intellij.util.ExceptionUtil.rethrowUnchecked(ExceptionUtil.java:143)
		at com.intellij.util.ExceptionUtil.rethrow(ExceptionUtil.java:158)
		at com.intellij.util.ConcurrencyUtil.manifestExceptionsIn(ConcurrencyUtil.java:329)
		at com.intellij.codeInsight.daemon.impl.DaemonCodeAnalyzerImpl.lambda$queuePassesCreation$4(DaemonCodeAnalyzerImpl.java:1381)
		at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:246)
		at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
		at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:208)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:343)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:341)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:710)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1098)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:652)
		at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:307)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:650)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:199)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
		at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1$0(NonBlockingFlushQueue.kt:329)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:941)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1(NonBlockingFlushQueue.kt:328)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:325)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:753)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:321)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:158)
		at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
		at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:731)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:384)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1122)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1122)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:382)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:422)
		at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
		at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: com.intellij.util.ExceptionUtilRt$RethrownStack: Rethrown at
		at com.intellij.util.ExceptionUtilRt.addRethrownStackAsSuppressed(ExceptionUtilRt.java:43)
		at com.intellij.util.ExceptionUtilRt.rethrowUnchecked(ExceptionUtilRt.java:17)
		at com.intellij.util.ExceptionUtil.rethrowUnchecked(ExceptionUtil.java:143)
		at com.intellij.util.ExceptionUtil.rethrow(ExceptionUtil.java:158)
		at com.intellij.util.ConcurrencyUtil.manifestExceptionsIn(ConcurrencyUtil.java:329)
		at com.intellij.codeInsight.daemon.impl.DaemonCodeAnalyzerImpl.lambda$queuePassesCreation$4(DaemonCodeAnalyzerImpl.java:1381)
		at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:246)
		at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
		at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:208)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:343)
		at com.intellij.openapi.application.RunnableToLambda.invoke(coroutines.kt:341)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:710)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1098)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:652)
		at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:307)
		at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:650)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:199)
		at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
		at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1$0(NonBlockingFlushQueue.kt:329)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:941)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$1(NonBlockingFlushQueue.kt:328)
		at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:325)
		at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:753)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:321)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
		at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:158)
		at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
		at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:731)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:384)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1122)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1122)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:382)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:422)
		at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
		at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'mappings' of com/redhat/devtools/lsp4ij/LanguageServersRegistry.addServerDefinitionWithoutNotification must not be null
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.$$$reportNull$$$0(LanguageServersRegistry.java)
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.addServerDefinitionWithoutNotification(LanguageServersRegistry.java)
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.loadServersAndMappingsFromExtensionPoint(LanguageServersRegistry.java:216)
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.initialize(LanguageServersRegistry.java:97)
	at com.redhat.devtools.lsp4ij.LanguageServersRegistry.<init>(LanguageServersRegistry.java:91)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate$lambda$0(instantiate.kt:48)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate$lambda$3$0(instantiate.kt:294)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.withStoredTemporaryContext(instantiate.kt:310)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:293)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:45)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:33)
	... 52 more
```

--- a/src/main/java/com/redhat/devtools/lsp4ij/LanguageServersRegistry.java
+++ b/src/main/java/com/redhat/devtools/lsp4ij/LanguageServersRegistry.java
@@ -206,7 +206,7 @@ public class LanguageServersRegistry implements Disposable {
         for (ServerExtensionPointBean server : ServerExtensionPointBean.EP_NAME.getExtensions()) {
             String serverId = server.id;
             if (serverId != null && !serverId.isEmpty()) {
-                List<ServerMapping> mappingsForServer = mappings.get(serverId);
+                List<ServerMapping> mappingsForServer = mappings.getOrDefault(serverId, Collections.emptyList());
                 mappings.remove(serverId);
                 var serverDefinition = new ExtensionLanguageServerDefinition(server);
                 SemanticTokensColorsProvider semanticTokensColorsProvider = semanticTokensColorsProviders.get(serverId);

